docs/conf.py

```diff
@@ -109,7 +114,27 @@ def setup(app):
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This patterns also effect to html_static_path and html_extra_path
-exclude_patterns = ['_build', '**.ipynb_checkpoints', 'examples/experiments']
+exclude_patterns = [
+    '_build',
+    '**.ipynb_checkpoints',
+    'examples/experiments/edwardpyhf.ipynb',
+    'examples/notebooks/ImpactPlot.ipynb',
+    'examples/notebooks/ImpactPlot.ipynb',
```

Comment:
This line looks like it snuck in here twice given the line above.
```suggestion
```

Also, can these be added as just a directory instead of individual files? So just `examples/notebooks/`?